app/controllers/owners/menus_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Owners::MenusController < ApplicationController
+  def new
+    @menu = Menu.new
+    @restaurant = Restaurant.find(params[:restaurant_id])
+  end
+
+  def create
+    @menu = Menu.new(name: params[:menu][:name], restaurant_id: params[:restaurant_id])
```

Comment:
We updated the code with sanitising the params. However, is this needed as we in the previous code (above) only are looking for a specific params attribute?